data/lib/src/network/linshare_http_client.dart

```diff
@@ -62,6 +63,14 @@ class LinShareHttpClient {
     return PermanentToken.fromJson(resultJson);
   }
 
+  Future<bool> deletePermanentToken(Token token) async {
```

Comment:
the method param should be `PermanentToken` type. I know its the same as `Token` class. But I want to separate POJO model between `data` layer and `domain` layer. Then if we have changes from remote in the future, we only need to edit model class in `data` layer,  don't need to edit anything from `domain`. 
I already created an extension to convert between these, please use `token.toPermanentToken()`

Reply:
Sure, I used `PermanentToken` in Data Layer and `Token` in Domain, are you ok with this?